README.md

```diff
@@ -57,7 +59,7 @@ Thus, webpack CLI provides different commands for many common tasks.
 -   `build|bundle|b [entries...] [options]` - Run webpack (default command, can be omitted).
 -   [`configtest|t [config-path]`](./packages/configtest/README.md#webpack-cli-configtest) - Validate a webpack configuration.
 -   `help|h [command] [option]` - Display help for commands and options.
--   [`init|create|new|c|n [generation-path] [options]`](./INIT.md#webpack-cli-init) - Create a new webpack project.
+-   [`init|create|new|c|n [generation-path] [options]`](./packages/init/README.md#webpack-cli-init) - Create a new webpack project.
```

Comment:
Init package was moved to generators. So the old link is good here.

Reply:
let's move the init readme to generators then. Following this logic, we would have had merge, bundle and every other command at root level too.